docker-image/v1.3/debian-cloudwatch/conf/kubernetes.conf

```diff
@@ -188,5 +188,8 @@
 <filter kubernetes.**>
   @type kubernetes_metadata
   @id filter_kube_metadata
+  kubernetes_url "#{ENV['FLUENT_FILTER_KUBERNETS_URL'] || https://#{ENV['KUBERNETES_SERVICE_HOST']}:#{ENV['KUBERNETES_SERVICE_PORT']}/api}"
```

Comment:
I think this won't work.
How do you feel like the following code?
` "#{ENV['FLUENT_FILTER_KUBERNETS_URL'] || "https://#{ENV.fetch('KUBERNETES_SERVICE_HOST')}:#{ENV.fetch('KUBERNETES_SERVICE_PORT')}/api"}"`

Reply:
That's fine with me too. Updated the PR.